docker-compose.yaml

```diff
@@ -39,6 +39,8 @@ services:
     restart: always
     expose:
     - "8080"
+    ports:
```

Comment:
These ports are closed in the base file for security reasons.
Any port opening should be done in the flavour-specific (e.g. `datagov`) files.